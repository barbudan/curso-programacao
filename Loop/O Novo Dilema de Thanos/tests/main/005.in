15
1
2
3
4
5
6
7
8
9
10
1000000000008
1000000000007
1000000000006
1000000000005
1000000000004