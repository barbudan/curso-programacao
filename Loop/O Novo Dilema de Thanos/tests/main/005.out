Thanos poupara o Planeta 1.
Thanos nao poupara o Planeta 2.
Thanos poupara o Planeta 3.
Thanos nao poupara o Planeta 4.
Thanos poupara o Planeta 5.
Thanos nao poupara o Planeta 6.
Thanos poupara o Planeta 7.
Thanos nao poupara o Planeta 8.
Thanos poupara o Planeta 9.
Thanos nao poupara o Planeta 10.
Thanos nao poupara o Planeta 11.
Thanos poupara o Planeta 12.
Thanos nao poupara o Planeta 13.
Thanos poupara o Planeta 14.
Thanos nao poupara o Planeta 15.
